lua/trunk.lua

```diff
@@ -226,6 +257,30 @@ end
 
 -- Startup, including attaching autocmds
 local function start()
+	local cliVersion = getCliVersion()
+	logger.info("Running on CLI version:", cliVersion)
+	if cliVersion == nil then
+		logger.error("nil CLI version")
+		print(
+			"The Trunk Neovim extension requires Trunk CLI version >= 1.17.0 - we could not determine your Trunk CLI version"
+		)
+		print("The extension will not run until you upgrade your CLI version")
+		print("Please run `trunk upgrade cli` to upgrade your CLI version")
+		return
+	end
+	if not checkCliVersion("1.17.0") then
+		logger.error("Trunk CLI version must be >= 1.17.0")
+		print("The Trunk Neovim extension requires Trunk CLI version >= 1.17.0 - you currently have " .. cliVersion)
+		print("The extension will not run until you upgrade your CLI version")
+		print("Please run `trunk upgrade cli` to upgrade your CLI version")
```

Comment:
Change these to `trunk upgrade`. Strictly speaking this is correct, but we want to recommend `trunk upgrade` as a better practice since it includes plugin fixes.